docs/Testing.TestingAndDebugging.md

```diff
@@ -61,7 +61,7 @@ The response then contains the **searchQueryStr** attribute, as follows:
     				"parent" : "/permid.org"
       			}
     		} ],
-    "searchQueryStr" : "{\n  \"from\" : 0,\n  \"size\" : 1,\n  \"query\" : {\n\"filtered\" : {\n  \"query\" : {\n\"bool\" : {\n  \"must\" : {\n\"match\" : {\n  \"CommonName.mdaas\" : {\n\"query\" : \"Disney\",\n\"type\" : \"phrase\"\n  }\n}\n  }\n}\n  },\n  \"filter\" : {\n\"bool\" : {\n  \"must\" : [ {\n\"term\" : {\n  \"parent\" : \"/permid.org\"\n}\n  }, {\n\"range\" : {\n  \"lastModified\" : {\n\"from\" : null,\n\"to\" : null,\n\"include_lower\" : true,\n\"include_upper\" : true\n  }\n}\n  } ]\n}\n  }\n}\n  },\n  \"fields\" : [ \"type\", \"system.path\", \"system.uuid\", \"system.lastModified\", \"content.length\", \"content.mimeType\", \"linkTo\", \"linkType\", \"system.dc\", \"system.indexTime\", \"system.quad\" ],\n  \"sort\" : [ {\n\"system.lastModified\" : {\n  \"order\" : \"desc\"\n}\n  } ]\n}"
+    "searchQueryStr" : "{\n  \"from\" : 0,\n  \"size\" : 1,\n  \"query\" : {\n\"filtered\" : {\n  \"query\" : {\n\"bool\" : {\n  \"must\" : {\n\"match\" : {\n  \"CommonName.mdaas\" : {\n\"query\" : \"Disney\",\n\"type\" : \"phrase\"\n  }\n}\n  }\n}\n  },\n  \"filter\" : {\n\"bool\" : {\n  \"must\" : [ {\n\"term\" : {\n  \"parent\" : \"/permid.org\"\n}\n  }, {\n\"range\" : {\n  \"lastModified\" : {\n\"from\" : null,\n\"to\" : null,\n\"include_lower\" : true,\n\"include_upper\" : true\n  }\n}\n  } ]\n}\n  }\n}\n  },\n  \"fields\" : [ \"type\", \"path.system\", \"uuid.system\", \"lastModified.system\", \"length.content\", \"mimeType.content\", \"linkTo\", \"linkType\", \"dc.system\", \"indexTime.system\", \"quad.system\" ],\n  \"sort\" : [ {\n\"lastModified.system\" : {\n  \"order\" : \"desc\"\n}\n  } ]\n}"
```

Comment:
nope.
This is actual request string sent internally to ES.
don't edit manually. try the search with `debug-info` flag, and copy it as is.
(P.S. in ES ns comes first. always...)

Reply:
Gilad, I ran this query just now on the Piranha version in production, and I'm still getting the system tag before the field name (e.g. system.indexTime). Any thoughts? What do you want me to put in documentation?

Reply:
Of course. As said, debug-info string is the exception. Dont switch the system namespace there. This hunk's previous version before current change is the correct one.

Reply:
Got it. Fixed now.